src/tiled/abstractobjecttool.cpp

```diff
@@ -23,6 +23,7 @@
 #include "addremovetileset.h"
 #include "changemapobject.h"
 #include "documentmanager.h"
+#include "fileformat.h"
```

Comment:
Include not needed.

Reply:
This uses ``FormatHelper`` which is now from ``fileformat.h``. It will compile without it explicitly included though because doubtless something in the chain pulls it in. In this situation I favour an explicit include but I'm happy to defer to you. Please confirm that you definitely want it changed.

Reply:
In that case let's keep it. Generally I consider includes superfluous when a subclass is already included (like `tmxmapformat.h` in this case), but since `FormatHelper` is another hierarchy we can keep it.

Reply:
Ok sounds good :+1: